### PR TITLE
feat: eve-connector priorityclassname added

### PR DESCRIPTION
**Pull request by a1ex-var1amov**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [x] Chart Version bumped for the respective charts
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [x] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/node-analyzer/README.md [hunk 1]
```diff
@@ -187,6 +187,7 @@ The following table lists the configurable parameters of the Sysdig Node Analyze
...
 | `nodeAnalyzer.runtimeScanner.eveConnector.resources.limits.memory`   | Eve Connector Memory limits per node                                                     | `512Mi`                                                                                                                                                               |
 | `nodeAnalyzer.runtimeScanner.eveConnector.settings.replicas`         | Eve Connector deployment replicas                                                        | `1`                                                                                                                                                                   |
+| `nodeAnalyzer.runtimeScanner.eveConnector.priorityClassName`         | Eve Connector deployment Priority class name                              | `{}`                                                                                                                                                               |
```
Comment (aroberts87) [inline charts/node-analyzer/README.md]:
```suggestion
| `nodeAnalyzer.runtimeScanner.eveConnector.priorityClassName`         | Name of an existing PriorityClass to use for the Eve Connector Deployment                              | `{}`                                                                                                                                                               |
```

File: charts/node-analyzer/templates/runtimeScanner/eveconnector-api-deployment.yaml [hunk 2]
```diff
@@ -28,6 +28,9 @@ spec:
 {{ include "nodeAnalyzer.labels" . | indent 8 }}
     spec:
       serviceAccountName: {{ template "nodeAnalyzer.serviceAccountName" .}}
+      {{- if .Values.nodeAnalyzer.runtimeScanner.eveConnector.priorityClassName }}
+      priorityClassName: "{{ .Values.nodeAnalyzer.runtimeScanner.eveConnector.priorityClassName }}"
```
Comment (aroberts87) [inline charts/node-analyzer/templates/runtimeScanner/eveconnector-api-deployment.yaml]:
```suggestion
      priorityClassName: {{ .Values.nodeAnalyzer.runtimeScanner.eveConnector.priorityClassName | quote }}
```

**Review by aroberts87 — CHANGES_REQUESTED** [review]
The changes look good, though I've noted some possible improvements. In particular I'd recommend changing the description of the new parameter to indicate that the chart will not create a new PriorityClass with the name given, but will use an existing one instead.

The only required change here is bumping the value of `version` in the node analyzer chart's [Chart.yaml](https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/Chart.yaml#L6)

**Comment by a1ex-var1amov** [comment]
> The changes look good, though I've noted some possible improvements. In particular I'd recommend changing the description of the new parameter to indicate that the chart will not create a new PriorityClass with the name given, but will use an existing one instead.
> 
> The only required change here is bumping the value of `version` in the node analyzer chart's [Chart.yaml](https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/Chart.yaml#L6)

version bumped! thank you!

**Review by aroberts87 — APPROVED** [review]

**Pull request merged**